src/App.vue

```diff
@@ -1,24 +1,30 @@
 <template>
-  <div id="app" :class="this.$store.getters.theme">
-    <Navbar />
-    <Settings />
-    <About />
-    <router-view/>
+  <div id="app" class="theme-dark h-screen w-full">
+    <Navbar class="card " />
+    <!-- <Setting s />
```

Comment:
could i ask why you're nuking the modals? also, could you please resolve the import issue for dark, cause it's kind of hard to evaluate if this is working without any deployment... and you're basically nuking the entire point of the theme so we'd have to swap the class manually, which our setup was already attempting to avoid...